e2e/e2e_test.go

```diff
@@ -1,5 +1,6 @@
 // +build e2e
 
+//
```

Comment:
Unsure if you were maybe planning to write a comment here?

Reply:
I wanted to write a high-level summary of what the file tests, but didn't come up with one and left this on accident. Thanks will remove